net.blumia.pineapple.steam-recording-exporter.yml

```diff
@@ -0,0 +1,89 @@
+app-id: net.blumia.pineapple.steam-recording-exporter
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: PineappleSteamRecordingExporter
+rename-icon: net.blumia.pineapple.steam-recording-exporter
+
+finish-args:
+  # X11 and Wayland support
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  
+  # Audio/Video support
+  - --socket=pulseaudio
+  - --device=dri
+  
+  # File system access
+  - --filesystem=home:ro
+  - --filesystem=~/.local/share/Steam:ro
+  - --filesystem=~/.var/app/com.valvesoftware.Steam/.local/share/Steam
+  - --filesystem=~/.steam/steam:ro
+  - --filesystem=~/Videos:rw
+  - --filesystem=~/Movies:rw
```

Comment:
given you grant excessive access to home you wouldn't need any of the others.

But then why do you need these?

Reply:
Oh, see the comment below for why: https://github.com/flathub/flathub/pull/7007#issuecomment-3369830044

So the suggested way is keep `-filesystem=home:ro` and remove others? Or remove `-filesystem=home:ro` and keep others?

Reply:
PR updated to go with keep `-filesystem=home:ro`.

Reply:
I'm not sure if I understand it correctly. After change it to code like below as you suggested, it then cannot detect Steam as intended:

```
  # File system access
  - --filesystem=home:ro
  - --filesystem=~/Videos:rw
  - --filesystem=~/Movies:rw
```

Thus I have to change it back.

Reply:
> This app requires access to recording files that are located in Steam's game recording folder, thus it requires read-only access to Steam's data path.

So ask for that only? Why do you have entire home and other directories here?

Reply:
> So ask for that only? Why do you have entire home and other directories here?

The home one is actually not needed but a "just in case" one. Since according to `xdg-user-dir` the user's video directory might be using a localized name like `~/视频` which will not covered by `~/Videos` or `~/Movies`. I can remove the `home` one if this can be resolved in another way.